project/Deps.scala

```diff
@@ -209,8 +209,10 @@ object Deps {
     Test.pgEmbedded
   )
 
-  def appCommons(scalaVersion: String) =
+  val appCommons =
     List(
+      Compile.akkaHttp,
```

Comment:
We need to be careful with this, we don't want to introduce akka to everything that doesn't already depend on it. This is introducing akka to our `wallet`, `cli`, etc.